.pylintrc

```diff
@@ -0,0 +1,503 @@
+[MASTER]
```

Comment:
I generated this file with this: `pylint --generate-rcfile > .pylintrc`. Then I edited the `disabled=` and `enabled=` to be what you see in this PR.